docs/source/tutorial/calc_tutorial.rst

```diff
@@ -64,12 +71,34 @@ digital camera mounted onto a microscope.  We defined our detector as a 100 x
 computation time. The ``spacing`` argument tells HoloPy how far apart each
 pixel is. Both parameters affect the absolute size of the detector.
 
+Finally, we need to specify the *scattering theory* which knows how to calculate the hologram from the experimental setup and the scatterer:
```

Comment:
I don't think scattering theories should be central to the discussion here, in an effort to keep things simple. Part of the purpose of HoloPy is to abstract away the need for any optics knowledge so that holography can be more widely used as a general purpose tool. Of course we do need a discussion of scattering theories for more advanced users, but I don't think it belongs in the default case.